src/Commands/ExportPostmanCommand.php

```diff
@@ -77,24 +77,34 @@ public function handle(): void
                 }
 
                 if ($this->config['enable_formdata']) {
-                    $rulesParameter = null;
+                    $rulesParameter = collect($reflectionMethod->getParameters())
+                        ->filter(function ($value, $key) {
+                            $value = $value->getType();
 
-                    foreach ($reflectionMethod->getParameters() as $parameter) {
-                        if (! $parameterType = $parameter->getType()) {
-                            continue;
-                        }
-
-                        $requestClass = $parameterType->getName();
+                            return $value && is_subclass_of($value->getName(), FormRequest::class);
+                        })
+                        ->first();
 
-                        if (is_subclass_of($requestClass, FormRequest::class)) {
-                            $rulesParameter = new $requestClass();
-                        }
+                    if (! $rulesParameter) {
+                        continue;
                     }
 
-                    if ($rulesParameter instanceof FormRequest) {
-                        $requestRules = $rulesParameter->rules();
+                    $rulesParameter = $rulesParameter->getType()->getName();
+                    $rulesParameter = new $rulesParameter;
+
+                    $requestRules = [];
+                    $rules = method_exists($rulesParameter, 'rules') ? $rulesParameter->rules() : [];
 
-                        $requestRules = array_keys($requestRules);
+                    foreach ($rules as $fieldName => $rule) {
+                        $requestRules[] = $fieldName;
+
+                        if (is_string($rule)) {
+                            $rule = preg_split('/\s*\|\s*/', $rule);
+                        }
```

Comment:
What is this for? Could it lead to unexpected behaviors?

Reply:
@tomirons this is if a rule is defined like string.. eg:
```php
'password' => 'required|string|max:8|confirmed'
```
It has to be parsed to array.

Reply:
to handle all unnecessary potential whitespace around the rule itself, eg:
```
'field' => 'rule_one |   rule_two'
```

Reply:
if it bugs you i can replace it with simple explode()